### ISSUE TITLE
Readme: Resolver configuration.

### ISSUE THREAD

**Issue by slawus** [oscarotero/Embed#67]
Hi! I found (correct me if I'm wrong) mistake in readme, in section 'Customization -> The request resolvers'. It says that configuration on resolver should look like this:
```php
$config = [
    'resolver' => [
        'class' => 'Embed\\RequestResolvers\\Curl' //The default resolver used

        'config' => [
            CURLOPT_MAXREDIRS => 20,
            CURLOPT_CONNECTTIMEOUT => 10,
            CURLOPT_TIMEOUT => 10,
            CURLOPT_SSL_VERIFYPEER => false,
            CURLOPT_SSL_VERIFYHOST => false,
            CURLOPT_ENCODING => '',
            CURLOPT_AUTOREFERER => true,
            CURLOPT_USERAGENT => 'Embed PHP Library',
            CURLOPT_IPRESOLVE => CURL_IPRESOLVE_V4,
        ]
    ]
];
```

But it looks like Embed::create uses $config['request]['resolver'] and $config['request']['config']  [code](https://github.com/oscarotero/Embed/blob/master/src/Embed.php).

Am i right? Imho version from readme is cleaner.

**Comment by oscarotero** [comment]
Hi.
Yes, you're right.
I've made it as was defined in the readme but I suspect my dog changed it while I slept :dog: 
Thanks for the feedback!!

**Comment by oscarotero** [comment]
Fixed in [2.4.3](https://github.com/oscarotero/Embed/releases/tag/v2.4.3). Thanks again :)

**Comment by slawus** [comment]
Yeah.. these dogs :) No problem, thanks for great library!
